components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -145,7 +148,7 @@
 <config_Redi_N2_based_taper_enable>.true.</config_Redi_N2_based_taper_enable>
 <config_Redi_N2_based_taper_min>0.1</config_Redi_N2_based_taper_min>
 <config_Redi_N2_based_taper_limit_term1>.true.</config_Redi_N2_based_taper_limit_term1>
-<config_Redi_min_layers_diag_terms>6</config_Redi_min_layers_diag_terms>
+<config_Redi_min_layers_diag_terms>15</config_Redi_min_layers_diag_terms>
```

Comment:
If I undrestand correctly, this is changing Redi for all configurations that have GM on.  This needs to only be for `ARRM10to60E2r1`.  I believe this is correct:
``` xml
<config_Redi_min_layers_diag_terms>6</config_Redi_min_layers_diag_terms>
<config_Redi_min_layers_diag_terms ocn_grid="ARRM10to60E2r1">15</config_Redi_min_layers_diag_terms>
```

Reply:
Oops, that was definitely my mistake commit, I certainly should have included the grid specification. Thanks for the catch!
@milenaveneziani I just want to confirm that we do indeed want this separate setting for this mesh, before I make the change (or just revert the commit)?

Reply:
yes, we want it at 15 for this mesh.